node/resolvers/catalog/calculatedAttachmentsResolver.ts

```diff
@@ -0,0 +1,237 @@
+import http from 'axios'
+import { propEq, prop, head, find, map, pluck, isEmpty, pickBy, both } from 'ramda'
+import { renameKeysWith } from 'ramda-adjunct'
+import camelCase from 'lodash.camelcase'
```

Comment:
We avoid having more than one general purpose library. It will be better used only Ramda

Reply:
Ramda has no function to rename keys and camel casing, I'll fork these functionalities over to store-graphql